types/stylelint/index.d.ts

```diff
@@ -1,541 +1,620 @@
 declare module 'stylelint' {
```

Comment:
[note] The deletion of `namespace` causes indent changes in many lines. I recommend the diff by [hiding whitespace mode](https://github.com/stylelint/stylelint/pull/6510/files?w=1).